src/FaceFactory.ts

```diff
@@ -0,0 +1,36 @@
+import { Hash, hashFactory, sumAndDiff } from './';
+
+export interface Face {
+  color: string;
+  eyes: string;
+  nose: string;
+  mouth: string;
+}
+
+export class FaceFactory {
```

Comment:
So I think that we can make FaceFactory generic on T, the same way that Hash is; you can pass an array of any arbitrary type and receive back one of those things. That and a test for two different types of input would be great 👍

Reply:
Sure! Now FaceFactory accepts arrays of different types. I added tests for integers and objects.